### DISCUSSION TITLE
error: globalThis is not defined

### DISCUSSION THREAD

**Discussion by MrakoMaks** [requarks/wiki#4411] [Error / Bug Report] [answered]
Good afternoon! When I try to launch a new version, I get the following error:

![image](https://user-images.githubusercontent.com/84027393/132990997-f5147140-0fc8-4c35-81f3-8fc01c187c1f.png)

**Comment by MrakoMaks** [comment, marked as answer]
The problem was solved banally. It was necessary to update node.js

**Reply by Gond0r** [reply]
What version of Node.js did you upgrade to?

**Reply by MrakoMaks** [reply]
v14.17.6

**Comment by Stryt** [comment]
Hello,

I have solved this problem to for "node server" command, but in the "journalctl" error appears yet.

![image](https://user-images.githubusercontent.com/15654608/133260375-ff6fee65-1312-48ce-8fdf-c9a79ec1f75d.png)

Anyone have the same problem?

Thanks

*Problem solved with another post on forum.
thanks again :)

**Reply by Gond0r** [reply]
Would you please share that forum post URL?

**Reply by ameyrakheja** [reply]
I upgraded from 2.5.201 to 2.5.296, and upgraded node.js as well from 9.3.1 to 14.19.0. However, my systemctl service didn't reference the new node.js install path, so changing `ExecStart = /usr/bin/node server` to `ExecStart=/usr/local/bin/node server` fixed it for me.
